scholarx/2021/functions.js

```diff
@@ -103,8 +103,13 @@ function loadFeaturedStories() {
         dataType: 'json',
         success: function (data) {
             document.getElementById('btnFeaturedStories').style.visibility = "visible";
+            
+            //hide section if there are no featured Stories
+            if (data.length == 0){
+              document.getElementById("featured-stories-section").style.display = "none";
+            }
```

Comment:
I think this way would be better
```suggestion
            
            //hide section if there are no featured Stories
            if (data.length == 0){
              document.getElementById("featured-stories-section").style.display = "none";
            } else if (data.length >=6) {
                //slice array to two parts
                let partOne = data.slice(0, 6);
```

Reply:
Hello @Piumal1999 thank you for the feedback. Are you suggesting that I bring up the `else if` block near the closing bracket of the `if` block. Because I can't seem to find any syntactical difference in your suggestion apart from that. If there are, please be kind enough to let me know. Thank you :)

Reply:
> Are you suggesting that I bring up the else if block near the closing bracket of the if block.

yes exactly

Reply:
@Piumal1999 what would happen if I commit suggestion as follows:
![image](https://user-images.githubusercontent.com/49517852/132118510-1f6837fa-9124-4a43-9641-d517f7efe325.png)
Or do I need to make changes and ammend?

Reply:
Make the changes and amend. Commit suggestion button will not do the required change in this case

Reply:
please take a look now :)

Reply:
@Piumal1999